frontend/packages/core/src/AppLayout/user.tsx

```diff
@@ -76,34 +102,52 @@ const UserInformation: React.FC = () => {
   }
 
   return (
-    <Grid container alignItems="center" justify="flex-end">
-      <div>{userId()}</div>
+    <Box>
       <UserPhoto
         ref={anchorRef}
         edge="end"
         aria-controls={open ? "account-options" : undefined}
         aria-haspopup="true"
-        onClick={handleToggle}
+        onMouseEnter={handleToggle}
       >
-        <Avatar>
-          <Initials variant="h6">{userId().slice(0, 2).toUpperCase()}</Initials>
-        </Avatar>
+        <UserAvatar />
       </UserPhoto>
-      <Popper open={open} anchorEl={anchorRef.current} role={undefined} transition>
+      <Popper
+        open={open}
+        anchorEl={anchorRef.current}
+        role={undefined}
+        transition
+        popperOptions={{
+          modifiers: {
+            offset: {
+              offset: "-105,0",
```

Comment:
the popper has this css property `transform: translate3d` which sets its position. i changed where the popper ends so that it better matches the figma design

https://stackoverflow.com/questions/60122232/how-do-you-position-material-ui-popper-in-the-bottom-right-corner-of-the-browser